python/triton/compiler.py

```diff
@@ -1013,6 +1011,9 @@ def _compile(fn, signature: str, device: int = -1, constants=dict(), specializat
     if output == "ptx":
         return ptx, shem_size, kernel_name
 
+    with open('triton.ptx', 'w') as f:
```

Comment:
Is this for debug?

Reply:
Yes, it's for debugging only. Sorry this (draft) branch was not ready for review yet.

Reply:
Resolved